tests/fixture/database/test_redis.py

```diff
@@ -0,0 +1,173 @@
+import pytest
+
+from pytest_mock_resources import create_redis_fixture
+
+redis = create_redis_fixture()
+
+
+def _sets_setup(redis):
+    redis.sadd("friends:leto", "ghanima")
+    redis.sadd("friends:leto", "duncan")
+    redis.sadd("friends:paul", "duncan")
+    redis.sadd("friends:paul", "gurney")
+
+
+def _hash_setup(redis):
+    redis.hset("user", "name", "foo")
+    redis.hset("user", "age", 30)
+
+
+def _list_setup(redis):
+    redis.lpush("dbs", "mongo")
+    redis.lpush("dbs", "redis")
+    redis.lpush("dbs", "postgres")
+    redis.lpush("dbs", "mysql")
+    redis.lpush("dbs", "mysql_lite")
+
+
+@pytest.mark.redis
```

Comment:
would you mind adding a test for pmr_credentials manual client construction from `.as_redis_kwargs()` as well as `Redis(pmr_credentials.as_url())`?